src/cli/cli.cpp

```diff
@@ -277,6 +277,26 @@ otError Interpreter::ParseJoinerDiscerner(char *aString, otJoinerDiscerner &aDis
     return error;
 }
 
+otError Interpreter::ParseIp6Prefix(char *aString, otIp6Prefix &aPrefix)
+{
+    otError       error = OT_ERROR_NONE;
+    char *        prefixLengthStr;
+    unsigned long length;
+
+    prefixLengthStr = strchr(aString, '/');
+    VerifyOrExit(prefixLengthStr != nullptr, error = OT_ERROR_INVALID_ARGS);
```

Comment:
use default 64 if not specified?

Reply:
I think it may be better to require the length to be given explicitly here and not use a default value (since this is used for an IPv6 prefix (not an Pv6 address)).

Reply:
Fine to be more strict.